### DISCUSSION TITLE
CIS CDN server | Discord ban in Russia

### DISCUSSION THREAD

**Discussion by TimmekHW** [mikotoIO/mikoto#568] [Ideas]
EN:
After blocking the Discord in Russia, we want to use mikoto.io

We need:
1. The interface is in Russian (I can personally help with the translation t.me/timmek1)
2. CDN in Russia. Preferably in the city of Kazan (tell me what your server requirements are, maybe I can help or rent for you, for you) 
3. Desktop APP for Windows 11 and for Astra Linux (Debian)

_________________________________________________________

RU:
После блокировки Дискорда в России мы хотим пользоваться mikoto.io

Нам нужно:
1. Интерфейс на русском языке (могу лично помочь с переводом t.me/timmek1)
2. CDN в России. Желательно в городе Казань (скажите какие у вас требование к серверам, возможно я смогу помочь или арендовать за вас, для вас) 
3. Приложение для компьютера для Windows 11 и для Astra Linux (Debian)

**Comment by TheCactusBlue** [comment]
Thank you very much for the suggestion!

1. Translations can be added in the `apps/client/public/locales` directory.
2. Mikoto can't have a server locally within Russia, due to sanctions and privacy concerns. However, we plan to provide tools to ensure that everyone can connect to the global server - if you're interested in hosting an unaffiliated Mikoto server, LMK!
3. The desktop app is on hold for now to focus on improving stability for the web app. If you want to contribute, please ask.

**Reply by TimmekHW** [reply]
1. Residents of the CIS countries prefer to see Russian instead of English. 
for many it is a second language. Can you add it to the site as well https://alpha.mikoto.io / ?

[translation.json](https://github.com/user-attachments/files/17311527/translation.json)


2. Sanctions can only be imposed by the UN. There are restrictions from the European Union and the United States. But as I see it, your headquarters is in Australia. Australia has blacklisted only state-owned corporations and individuals (I am not on this list).  And the Discord servers in Moscow continue to function. The blocking was initiated by government. Steam, Riot Games, and Google continue to have their servers in Russia. 

2.1. I can rent a dedicated server for mikoto and pay for it myself. Tell me what configuration is needed.

3. You can make an application based on the Edge engine. We want the app because we need "Noise Gate", KRISP or its more open counterpart. And it is also impossible to share the screen in the game "VALORANT" through a Web browser. There's an anti-cheat that doesn't allow you to do this through the browser

**Comment by darkwiz666** [comment]
@TheCactusBlue I suggest you close this 'discussion', he's only making demands for things he doesn't understand at this point, and this is for feature requests, bug fixes, and anything directly related to development. Anything else is just a distraction...and annoying.

**Reply by TimmekHW** [reply]
I suggest renting a server in Russia, for lower latency. I am ready to rent it at my own expense so that I and other users from the CIS have a pleasant experience of using it. Tell me the minimum configuration that will allow you to handle about 100 people without problems. I am ready to rent this server and transfer it to you. I'm not making an empty sound, I want to help you, myself and others like me
